src/common/util.h

```diff
@@ -129,6 +129,16 @@ void util_set_alloc_funcs(
 #define force_inline __attribute__((always_inline)) inline
 #endif
 
+
+#if (defined(__GNUC__) && !defined(__clang__))
+#define GCC_VERSION (__GNUC__ * 10000 + __GNUC_MINOR__ * 100 \
+					+ __GNUC_PATCHLEVEL__)
+
+#if GCC_VERSION >= 80101 /* >= 8.1.1 */
```

Comment:
Please don't hardcode version numbers (eg. what will happen when they fix it later?). Detect it based on behavior.